sql/updates/latest-dev.sql

```diff
@@ -118,3 +118,82 @@ ALTER TABLE IF EXISTS _timescaledb_catalog.compression_chunk_size ADD COLUMN IF
 CLUSTER  _timescaledb_catalog.compression_chunk_size USING compression_chunk_size_pkey;
 ALTER TABLE _timescaledb_catalog.compression_chunk_size SET WITHOUT CLUSTER;
 
+-- If the dimension slice table is broken and there are dimension
+-- slices missing from the table, we will repair it by:
+--    1. Finding all chunk constraints that have missing dimension
+--       slices and extract the constraint expression from the associated
+--       constraint.
+--    2. Parse the constraint expression and extract the column name,
+--       and upper and lower range values as text.
+--    3. Use the column type to construct the range values (UNIX
+--       microseconds) from these values.
+
+-- View with all dimension slices that are mentioned in the
+-- chunk_constraint table but are missing from the dimension_slice
+-- table.
+CREATE VIEW missing_slices(dimension_slice_id, constraint_name, constraint_expr) AS
```

Comment:
Do we need these views, which later requires dropping them?

Isn't it more convenient to just use CTEs?

```
INSERT INTO _timescaledb_catalog.dimension_slice
WITH missing_slices AS (
  ... 
), unparsed_missing_slices AS (

), rebuilt_missing_slices AS (

) SELECT * FROM rebuilt_missing_slices;
```